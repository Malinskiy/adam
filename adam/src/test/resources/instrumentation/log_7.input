INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[0]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[1]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=2
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[1]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=2
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[2]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=3
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[2]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=3
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[3]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=4
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[3]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=4
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[4]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=5
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[4]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=5
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[5]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=6
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[5]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=6
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[6]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=7
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: numtests=7
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: test=test[6]
INSTRUMENTATION_STATUS: class=com.example.ParameterizedTest
INSTRUMENTATION_STATUS: current=7
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_RESULT: stream=

Time: 0.009

OK (7 tests)


INSTRUMENTATION_CODE: -1
