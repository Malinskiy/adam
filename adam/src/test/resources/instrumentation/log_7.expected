TestRunStartedEvent(testCount=7)
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[0]), metrics={})
TestStarted(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[1]))
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[1]), metrics={})
TestStarted(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[2]))
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[2]), metrics={})
TestStarted(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[3]))
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[3]), metrics={})
TestStarted(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[4]))
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[4]), metrics={})
TestStarted(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[5]))
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[5]), metrics={})
TestStarted(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[6]))
TestEnded(id=TestIdentifier(className=com.example.ParameterizedTest, testName=test[6]), metrics={})
TestRunEnded(elapsedTimeMillis=9, metrics={})
